channeldb/db.go

```diff
@@ -165,7 +166,10 @@ func Open(dbPath string, modifiers ...OptionModifier) (*DB, error) {
 
 	// Specify bbolt freelist options to reduce heap pressure in case the
 	// freelist grows to be very large.
-	bdb, err := kvdb.Open(kvdb.BoltBackendName, path, opts.NoFreelistSync)
+	bboltOpts := &bbolt.Options{
```

Comment:
Q: In the etcd PR `NoFreeListSync` is an `lncfg` option. I wonder if we ever intend to actually set this to false? And if no, then perhaps we could default it somewhere (walletdb?) such that it's less hurdle to later on remove them from the code.

Reply:
I think we want this to be user-configurable as some users have reported slow startup times on weak machines (mobile, rpi) and opt to not set `NoFreeListSync`.